kimchi/src/folding/error.rs

```diff
@@ -0,0 +1,126 @@
+use super::expressions::{FoldingExp, IntegratedFoldingExpr};
+use super::FoldingEnv;
+use crate::folding::expressions::{Degree, FoldingColumn};
+use crate::folding::EvalLeaf;
+use ark_ff::Field;
+
+pub(crate) fn eval_exp_error<'a, F: Field, E: FoldingEnv<F>>(
+    exp: &FoldingExp<F>,
+    env: &'a E,
+    side: bool,
```

Comment:
What is side used for?

Reply:
to choose between one instance/witness and the other